test/metabase/api/pivots.clj

```diff
@@ -1,4 +1,6 @@
 (ns metabase.api.pivots
+  ;; TODO -- I think we should roll this all in to `metabase.query-processor.pivot-test` or maybe a new namespace
+  ;; called `metabase.query-processor.pivot.test-util`
```

Comment:
yeah this is basically a `test-util` namespace, I wasn't sure quite how to name it